contrib/pureconfig/jvm/src/main/scala/eu/timepit/refined/pureconfig/error/PredicateFailedException.scala

```diff
@@ -0,0 +1,4 @@
+package eu.timepit.refined.pureconfig.error
+
+final case class PredicateFailedException(message: String)
```

Comment:
I was thinking about adding a special exception for refinement errors in #101 but didn't came around adding it yet to the core module. Are there any reasons against moving this later to the core module?